data/language/en-GB.txt

```diff
@@ -3599,6 +3599,9 @@ STR_6490    :Warning: Semi-compatible Park Version
 STR_6491    :This park was saved in a later version of OpenRCT2. Park is v{INT32} and requires at least v{INT32}.
 STR_6492    :This park was saved in an old version of OpenRCT2, and can not be opened with this version of OpenRCT2. Park is v{INT32}.
 STR_6493    :This park was saved in a later version of OpenRCT2, some data may be lost. Park is v{INT32} and requires at least v{INT32}.
+STR_6494    :Group by ride type
+STR_6495    :Group rides by ride types instead of showing each vehicle separately.
+STR_6496    :{WINDOW_COLOUR_2}{STRINGID}
```

Comment:
@oli414 why is this a separate string? In what scenario will it be different for other languages?

Reply:
It won’t, but it’s much easier to do it this way. (There are lots of strings like this.)

Reply:
> In what scenario will it be different for other languages?

Not sure, like you said the examples I looked at were done in the same way, which is why it's done like this.